File: lab4/flex/CMinus/samples/sample1.c
int i;
int j;

int gcd(int u, int v)
{	if (v==0) return u;
	else return gcd(v, u-u/v*v);
	/* u-u/v*v == u mod v */
}

void main(void)
{	int x; int y;
	/*x = input(x); y = input(y);*/
	output(gcd(x, y));
}

File: lab3/samples/sample1.py
#
  这是测试样例1
  一个if分支测试
#
input x;
if 0 < x:
  fact = 0;
  if(x>2):
    fact = 1;
    if(x>3):
      fact = 2
    end
  end
else:
  fact = 3;
  print(fact)
end
